### ISSUE TITLE
Notes/Questions on Conservation

### ISSUE THREAD

**Issue by rburghol** [HARPgroup/vahydro#401]
Standard practice outlined by state drought task force is to begin drought response reductions during declared drought warning conditions.  One criteria for potentially declaring drought warning is when flows fall below 10% non-exceedance threshold.  Some question the utility of conservation measures at early stages of drought, reasoning that it is an inconvenience at a time that the stream is not necessarily vulnerable.  There are some counter assertions to that notion:
1. Preservation of natural flow variation is inherently a positive thing, and greater flows during *moderate drought* conditions help to bolster aquatic communities to survive during the extreme drought conditions that sometimes follow these moderate drought conditions, i.e., when an extreme drought occurs, it is usually preceded by months or weeks of moderate drought conditions.
2. By enacting conservation measures across all water users during moderate drought, available water is therefore increased in the event that one or more systems in a watershed experience some unforseen complication that prevents them from being able to meet water demands.  For example, water quality problems in a reservoir, large infrastructure leaks that lead to unusually high losses, groundwater well failure, or metering problems that result in rapid unexpected drawdowns in reservoirs can all result in water systems being in a compromised position earlier in a drought than expected.  

For the above reasons, synchronized conservation measures during early stages of drought can allow systems that are experiencing unforeseen difficulties in meeting flowbys to have the potential for temporary variances to replenish depleted storage or to meet peak demands with direct withdrawals until a solution can be found or until drought conditions abate.

**Comment by durelles** [comment]
Stepping, taking a systems perspective to protect and minimize the potential for cascading effects. The question I ponder is how does the risk change and potential lost revenue for producers with a proactive/systems approach. How often are cascading effects in play, and can previous history inform thresholds that balance ecosystem & ag production, not to mention my green lawn?

**Comment by julieshortridge** [comment]
That's an interesting point Rob. I could see it going both ways - certainly for a system with storage (reservoirs) early conservation measures would be helpful in terms of providing a buffer against more extreme impacts later. In other cases, I could see it being a tradeoff between longer-term moderate impacts (both ecologically and economically for water users who face restrictions), and shorter-term acute impacts. It would be interesting to see how that balanced out. 

Also, Scotty how is your lawn still green?

**Comment by laljeet** [comment]
**Some question the utility of conservation measures at early stages of drought, reasoning that it is an inconvenience at a time that the stream is not necessarily vulnerable**

This a very commonly said and true when we think about it from a user perspective. A farmer will say this because once the seed is below the soil, they lose most of the control and only know their water demand and seek ways to fulfill it. But the farmer has control before planting. 

One thing that I often think is can our conservation practices be shifted from impact-based to user-based. 
The questions that arise are:

Can we shift the action for conservation to users when they have control rather than when the flow goes/starts to go down?
One small (maybe an irrelevant) example is changing /shifting planting dates based on historical, expected trends. The monsoon impacts my home state, so if farmers are given a warning of a week monsoon, they get to plan/prepare for water shortages. The dates of monsoon arrival in the region are also predicted, and that is helpful. Farmers are advised to plant accordingly that the peak water demands are during the arrival of the monsoon. These dates vary from June 5 to June 20 every year. 

A similar example is from Alabama, where users make water withdrawals in Feb and March for water to be used in July-August (especially La Nina years). But again, these practices are suggested well in advance, and farmers have control over them.

**Can we find something that can predict/ give an indication of low flows on a season basis for our region rather than saying now we have limitations on withdrawals as flows are receding?**

The above is a larger systems view, but the second question comes for short-term droughts, which vary year to year (the whole idea on which the Tidewater project is on) and within months.
One way to plan on the user end for this is short-term forecasts which we are trying. Another is the whole idea of reservoir use. Optimum size (a farmer has a 10-acre pond but needs 5 or vice versa) and use.
How much is the optimum need? Shall we really make a reservoir for the worst-case scenario?  How convenient is it for the frame rot to know this on a daily scale? One example: A simple board stuck in water gives a farmer looks at it and tells him what to do (attached). 

In summary, we can think about why users think conservation measures at the early stages of drought are inconvenient and work on that.  I might have gotten too far on this, but these were some thoughts.

![1](https://user-images.githubusercontent.com/55749191/129383535-7726b993-47fa-47cd-b495-212d7c2683a7.jpeg)
